src/components/HURUmap/SubcategoryHeader/index.js

```diff
@@ -0,0 +1,53 @@
+import { RichTypography } from "@commons-ui/core";
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  root: {
+    borderBottom: `solid 1px ${palette.divider}`,
```

Comment:
@gertie-sheshe I think this should be on description. So that when description is not defined, the border does not show too
![Screenshot from 2021-09-23 10-10-24](https://user-images.githubusercontent.com/7962097/134467632-10733b0a-6fb9-42d4-8242-ebec9f2505aa.png)
.